fetch.js

```diff
@@ -197,6 +219,8 @@ function DoWork() {
     UpdateSdkManualNotes();
     CheckLegacyApiGroupList(playFabUrl, tocObj);
     CheckSwaggerApiGroupList(playFabUrl, tocObj);
+    // CheckAzureApiSpecGroupList(playFabUrl, tocObj);
```

Comment:
I would say we should un-comment these so we can start seeing the process in motion sooner, but you can leave them commented out until you are ready to leverage them.